internal/firewall/delete.go

```diff
@@ -0,0 +1,84 @@
+package firewall
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func isDeleteInstruction(instruction string) bool {
+	fields := strings.Fields(instruction)
+	for i := 0; i < len(fields); i += 2 {
+		switch fields[i] {
+		case "-D", "--delete": //nolint:goconst
```

Comment:
we should make this doesn't delete a rule by line number; otherwise we should not try to find the rule line number from this.